packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/ios/Runner/Info.plist

```diff
@@ -60,10 +60,22 @@
         <key>CFBundleURLName</key>
         <string>$(CF_BUNDLE_URL_NAME)</string>
         <key>CFBundleURLSchemes</key>
-        <array>
+        <array>{{#enable_social_logins}}
+        <string>fb{app_id}</string>{{/enable_social_logins}}
         <string>$(CF_BUNDLE_URL_SCHEME)</string>
         </array>
         </dict>
-    </array>
+    </array>{{#enable_social_logins}}
+    <key>FacebookAppID</key>
+        <string>{app_id}</string>
+        <key>FacebookClientToken</key>
+        <string>{client_token}</string>
```

Comment:
From where those data comes from ?

Reply:
User needs to insert app name, app id and client token that he got after he register his app in facebook developer console.
I just added that to readme file with documentation where is all explained

Reply:
I meant something else. The Info.plist reads those variables, which is find. But where they can be set ?

Reply:
User needs to replace this {app_id} with his app id, {client_token} with client token that he gets from facebook console when register new app, its more like placeholders not variables. 
Maybe i should mark them differently?
Or I misunderstood question again?

Reply:
I'm afraid that you missed only very important concept in the [rx_bloc_cli](https://pub.dev/packages/rx_bloc_cli).

✅ [Build Flavors](https://flutter.dev/docs/deployment/flavors) - Support for multiple flavors (development, staging and production)

From what I see this is not incorporated into the current solution. Let me know if you need information on how to address this comment